src/Properties/Property.js

```diff
@@ -75,6 +75,18 @@ const Property = React.forwardRef(
           );
         return null;
       }
+      if (definition.includes('-theme-form-kind-')) {
```

Comment:
Rather than embed the type name in the identifier, I'm thinking we should put a bit more structure inside grommet-designer for how it handles these things. For example:
- keep '-theme-' as a value to indicate that the options should come from the theme
- add an object to Property indicating how to determine the options. For example:
```
const themeOptions = {
  hpe: {
    Button: {
      kind: ['toolbar', 'cta-primary', 'cta-alternate'],
    },
    Form: {
      kind: ['survey'],
    },
  },
  grommet: {},
};
```

Reply:
I kept `-theme-` but instead of using themeOptions I just looked directly at the theme object for button (similar to what was already there). I didn't create a themeOptions object because I couldn't find a convenient way to get the name of the theme but I could access the theme object. For Form I included survey for all themes because it is the base theme. Let me know if this works okay or if you would prefer I can go back and do the themeOptions object instead.

Reply:
good decisions!